src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -709,6 +709,8 @@ private void UpdateSpeciesDetail(Species species)
             species.FormattedNameBbCode, species.ID, species.Generation, species.Population, species.Colour.ToHtml(),
             string.Join("\n  ", species.Behaviour.Select(b => b.Key + ": " + b.Value)));
 
+        speciesDetailsLabel.ExtendedBbcode += "\n";
```

Comment:
Maybe this would be better in the switch below? That way if the next bit of info is not added then the line change is not added unnecessarily.

Reply:
So something like `speciesDetailsLabel.ExtendedBbcode += "\n" + TranslationServer.Translate(`

Reply:
But then we have to add a line break for each and every species (Multicellular, etc) later.

Reply:
Not really a problem, in most text appending cases that's what you need anyway because at the end of a section you don't know if a new section is going to be added or not.